locobot/agent/locobot_mover.py

```diff
@@ -244,3 +244,20 @@ def explore(self):
 
     def drop(self):
         return self.bot.open_gripper()
+
+    def get_obstacles_in_canonical_coords(self):
+        """get the positions of obtacles position in the canonical coordinate system
+        instead of the Locobot's global coordinates as stated in the Locobot
+        documentation: https://www.pyrobot.org/docs/navigation.
+
+        the standard coordinate systems:
+          Camera looks at (0, 0, 1),
+         its right direction is (1, 0, 0) and
+         its up-direction is (0, 1, 0)
+
+         return:
+         list[(x, z)] of the obstacle location in standard coordinates
+        """
+        cordinates_in_robot_frame = self.bot.get_map()
+        cordinates_in_standard_frame = [(-c[1], c[0]) for c in cordinates_in_robot_frame]
```

Comment:
use this instead to avoid duplication of this logic - https://github.com/facebookresearch/droidlet/blob/989963970d601bf23458a59656e14c536870d266/locobot/agent/locobot_mover_utils.py#L71

Reply:
changed the code accordingly